includes/specials/SpecialUndelete.php

```diff
@@ -328,11 +328,16 @@ function isDeleted() {
 	function undelete( $timestamps, $comment = '', $fileVersions = array(), $unsuppress = false ) {
```

Comment:
Worth considering, but this is core code and it's not within the current scope of the ticket.